index.js

```diff
@@ -5,7 +5,7 @@ module.exports = robot => {
     const github = await robot.auth(event.payload.installation.id);
 
     // Get all issues that aren't the new issue
-    const allIssues = await github.issues.getForRepo(context.repo());
+    const allIssues = await github.paginate(github.issues.getForRepo(context.repo()), issues => issues);
```

Comment:
I opened a PR in https://github.com/probot/probot/pull/112 to update the `paginate` method to have a default callback, so we can get rid of `issues => issues` here once that's merged.